installer/tue-install-impl.bash

```diff
@@ -745,12 +745,19 @@ for target in $targets
 do
     tue-install-debug "Main loop: installing $target"
     tue-install-target $target
+    if [ ! $? -eq 0 ]
+    then
+        tue-install-error "Installed target: '$target' doesn't exist (anymore)"
+        return 1
+    fi
 
     if [[ -d $TUE_INSTALL_TARGETS_DIR/$target && "$tue_cmd" == "install" ]]
     then
         # Mark as installed
         tue-install-debug "[$target] marked as installed after a successful install"
         touch $TUE_INSTALL_INSTALLED_DIR/$target
+    else
+        tue-install-debug "[$target] succesfully updated"
```

Comment:
I don't understand this

Reply:
Oh I missed that

Reply:
Done, but shouldn't the checking of the directory not happen before the tue-install-target? Then no checking of the error code is needed.